### PR TITLE
Faster version of Chipper

### PR DESCRIPTION
**Pull request by ajjimeno**

### CODE REVIEW

File: unstructured_inference/models/chipper.py [hunk 1]
```diff
@@ -457,6 +484,47 @@ def __call__(self, input_ids: torch.LongTensor, scores: torch.FloatTensor) -> to
...
+        return any(
+            i
+            for la in _calc_banned_tokens(
+                input_ids,
+                num_batch_hypotheses,
+                self.repetition_window,
+                cur_len,
+            )
+            for i in la
+            if i not in self.skip_tokens
```
Comment (cragwolfe) [inline unstructured_inference/models/chipper.py:524]: this is not easy to read, why not just iterate through nested for loops the old-fashioned way?

Comment (ajjimeno) [reply unstructured_inference/models/chipper.py]: The loop looks like this now.

        for banned_tokens in _calc_banned_tokens(
            input_ids,
            num_batch_hypotheses,
            self.repetition_window,
            cur_len,
        ):
            for token in banned_tokens:
                if token not in self.skip_tokens:
                    return True

        return False

Comment (cragwolfe) [reply unstructured_inference/models/chipper.py]: thanks!

**Review by LaverdeS — APPROVED** [review]
LGMT! It is considerably faster now (X3) and it also fixed some `UncategorizedText` remnants/extras during generation.

**Comment by cragwolfe** [comment]
Test instructions, how verified?

**Comment by LaverdeS** [comment]
> Test instructions, how verified?

```
%%time

doc = DocumentLayout.from_file("layout-parser-paper-fast.pdf", detection_model=model)
for i in range(len(docb.pages)):
    print(f"********** Page {i}")
    for element in doc.pages[i].elements:
        display(element.type)
        display(element.text)
```

**Pull request merged**